target/linux/bcm63xx/base-files/etc/board.d/01_leds

```diff
@@ -31,6 +31,11 @@ adb,pdg-a4001n-a-000-1a1-ax)
 	ucidef_set_led_netdev "wlan0" "WIFI" "green:wifi" "wlan0"
 	ucidef_set_led_usbdev "usb" "USB" "green:service" "1-1"
 	;;
+adb,pdg-a4101n-a-000-1a1-ae)
+	ucidef_set_led_netdev "lan" "LAN" "green:internet" "eth0.1"
+	ucidef_set_led_netdev "wan" "WAN" "green:dsl" "eth0.2"
```

Comment:
There's no eth0.2 in the configuration since all (4) ethernet ports are added to eth0.1.